src/evaluations/data/evaluation_configs.py

```diff
@@ -568,6 +569,47 @@ def construct_sketch_estimator_config_name(sketch_name, sketch_config,
 
 
 # Document the estimators.
+def _independent_set_estimator(sketch_epsilon=None, estimate_epsilon=None):
```

Comment:
Should there be a test for this?

Reply:
Added a unit test to check if this works. PTAL. Thanks!